lte/cloud/go/services/eps_authentication/servicers/ul.go

```diff
@@ -35,8 +35,16 @@ const (
 	qosProfilePriorityLevel           = 15
 	qosProfilePreemptionCapability    = true
 	qosProfilePreemptionVulnerability = false
+
+	defaultMaxUlBitRate uint64 = 2000000000
```

Comment:
This is 2 Gbps uplink and 4Gbps downlink per UE. Can we use the same defaults as [S1AP integ tests](https://github.com/magma/magma/blob/master/lte/gateway/python/integ_tests/s1aptests/s1ap_wrapper.py#L360-L361), i.e. DL: 200 Mbps and UL: 100 Mbps

Also note the default APN name is "magma.ipv4", instead of "oai.ipv4"

Reply:
I just took the default values from subscriberdb default configs (https://github.com/magma/magma/blob/master/lte/gateway/configs/subscriberdb.yml#L41-L43), to have parity with subscriberdb.

will update oai.ipv4

Reply:
Ah, I see! Let's keep it at 2 Gbps and 4 Gbps then.

The goal is to switch integration tests to EPS Auth path, by modifying EPS Auth service to run inside AGW dev VM for integration testing. Let's discuss more offline.